pkg/rc/farm.go

```diff
@@ -228,7 +229,7 @@ START_LOOP:
 				go func(id fields.ID) {
 					defer func() {
 						if r := recover(); r != nil {
-							err := util.Errorf("Caught panic in rc farm: %s", r)
+							err := util.Errorf("Caught panic in rc farm: %s\n%s", r, debug.Stack())
 							rcLogger.WithError(err).
```

Comment:
`util.WithError()` optionally tags the error with a filename and line number where the error was created (if the error implements `util.CallsiteError`). As written, this log line's tags will point to the "Caught panic..." message above. For a panic handler, would it make sense to log the location of the panic instead? I assume is what is actually interesting.